cypress/page/login.page.ts

```diff
@@ -0,0 +1,26 @@
+class LoginPage {
+  private addressText: string;
+  private passwordText: string;
+  private loginBtn: string;
+
+  constructor() {
+    this.addressText = "#email";
+    this.passwordText = "#passwd";
+    this.loginBtn = "button#SubmitLogin.button.btn.btn-default.button-medium";
+  }
+
+  public writeAddress(): void {
+    cy.get(this.addressText).type("aperdomobo@gmail.com");
```

Comment:
don't use the mail and password values directly, add them as method parameters and add them in the test file when you call loginPage.login(mail, pwd);

Reply:
resolved